src/App.js

```diff
@@ -5,13 +5,13 @@ import Header from './components/Header/Header';
 class App extends Component {
     render() {
         return (
-          <body>
+          <div>
```

Comment:
Why have you used 'div' instead of 'body' here?
If there is any special reason(s), then I would like to know that too and would implement the same in my projects.

Reply:
yeah because in react as you will be knowing in this project div id `root` is already inside `body` tag.
and the `render` must be wrapped inside a parent element. 
If you use `body` here then basically you have having two body tags. One in `index.html` and the other in render and thus this throws an error in console.